Packs/TrendMicroVisionOne/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Trend Micro Vision One",
     "description": "Trend Micro Vision One is a purpose-built threat defense platform that provides added value and new benefits beyond XDR solutions, allowing you to see more and respond faster. Providing deep and broad extended detection and response(XDR) capabilities that collect and automatically correlate data across multiple security layers\u2014email, endpoints, servers, cloud workloads, and networks\u2014Trend Micro Vision One prevents the majority of attacks with automated protection.",
     "support": "partner",
-    "currentVersion": "1.3.0",
+    "currentVersion": "2.0.2",
```

Comment:
How did we jump from 1.3.0 to 2.0.2? In `master` we're in version `2.0.1`. Let's fix this.